dacite/types.py

```diff
@@ -95,3 +99,7 @@ def is_generic_collection(type_: Type) -> bool:
 
 def extract_generic(type_: Type) -> tuple:
     return type_.__args__  # type: ignore
+
+
+def is_enum(type_: Type) -> bool:
+    return isclass(type_) and issubclass(type_, Enum)
```

Comment:
Maybe this should catch the `TypeError` instead of testing with `inspect.isclass()`?